docs/6-Troubleshooting/intro.md

```diff
@@ -0,0 +1,5 @@
+### Need any help?
+
+Kool is constantly evolving, but if you feel confused or lost on how to take full advantage of the tools Kool' has to offer, this section might be particularly useful. 
+
+Here you will find known fixes or workarounds for commom questions. Also, in the case that the information here contained doesn't help out you can join our community over [slack]()!
```

Comment:
we already have the `/slack` link alias, please get it with @dbpolito and update this to link to it properly.

Reply:
oooh thats cool. I didn't knew about the `/slack` link! Awesome!

Reply:
> updated it as instructed.